03_launch_mgmt_cluster.sh

```diff
@@ -264,6 +255,18 @@ function launch_cluster_api_provider_metal3() {
       kubectl scale -n metal3 deployment.v1.apps capm3-controller-manager --replicas 0
       nohup make run >> capm3.out.log 2>> capm3.err.log &
     fi
+
+    if [ "${BMO_RUN_LOCAL}" = true ]; then
+      touch bmo.out.log
+      touch bmo.err.log
+      kustomize build "$kustomize_overlay_path" | kubectl apply -f-
+      kubectl scale deployment metal3-baremetal-operator -n metal3 --replicas=0
```

Comment:
this would be the wrong namespace, and I am uncertain about the deployment name. Let's wait for the BMO and CAPM3 PR to be merged, then we'll come back to this. Once this is merged, we will focus on the last BMO PR that cleans things up